lib/nixos-system.nix

```diff
@@ -1,15 +1,14 @@
 { self, lib, home-manager }:
 
-{ modules ? [ ], specialArgs ? { }, ... } @ args:
-lib.nixosSystem (args // {
-  specialArgs = lib.mergeAttrs
+{ modules ? [ ], modulesArgs ? { }, ... } @ args:
```

Comment:
I think it'll be confusing to have them named `modulesArgs` and `specialArgs`. Why not `nixosSpecialArgs`, `hmSpecialArgs` and `globalSpecialArgs`?

Reply:
Applied.